interactions-transitions-assignment/part-1-bar-chart/us-emissions-interaction.html

```diff
@@ -1,10 +1,23 @@
 <html>
-    <head>
-        <title>US Air Quality</title>
-    </head>
-    <body>
-        <div id="chart-container"></div>
-        <script src="https://d3js.org/d3.v6.min.js"></script>
-        <script src="./us-emissions-interaction.js"></script>
-    </body>
+  <head>
+    <title>US Air Quality</title>
+  </head>
+  <body>
+    <div id="chart-container"></div>
+    <div id="controls">
+      <input type="radio" id="state" name="sort" value="state">
+      <label for="state">Sort By State</label>
+      <br>
+      <input type="radio" id="emissions-desc" name="sort" value="emissions-desc">
+      <label for="emissions-desc">Sort By Emissions Descending</label>
```

Comment:
I love your use of labels with the input here and the "for" specifications. Really makes it clear to read and is great from an accessibility standpoint